TravelBug/src/Contexts/Authcontext.jsx

```diff
@@ -0,0 +1,80 @@
+import { useContext, useState } from "react";
+import { createContext } from "react";
+import axios from "axios";
+
+const AuthContext = createContext();
+
+const AuthProvider = ({ children }) => {
+  const localStorageItems = JSON.parse(
+    localStorage.getItem("LoginCredentials")
```

Comment:
I had read somewhere that eg., LOGIN_CREDENTIALS would be a good naming convention when you are using local or session storage.